MAUI/PDF-Viewer/Form-Filling.md

```diff
@@ -251,6 +251,279 @@ private void PdfViewer_FormFieldValueChanged(object sender, FormFieldValueChange
+{
+    if (formField is TextFormField textBoxField)
+    {
+        foreach (var widget in textBoxField.Widgets)
+        {
+            widget. ForegroundColor = Colors.Aquamarine;
+        }
+    }
+} 
+} 
+{% endhighlight %}
+{% endtabs %}
+
+
+
+# Customize BorderColor for Form fields
+The BorderColor sets the color of the border around the form field. This is helpful when you want to customize the outline of a form field to make it stand out or fit a specific design in a PDF document programmatically.
+###  Customize BorderColor for Form fields  programmatically 
+You can add a new form fields with Customize the  BorderColor  the PDF document programmatically by creating an form fields 
+
+{% tabs %}
+{% highlight xaml %}
+<Grid> 
+    <Grid.RowDefinitions> 
+        <RowDefinition Height="Auto" /> 
+        <RowDefinition Height="*" /> 
+    </Grid.RowDefinitions> 
+    <Button Text="AddBorderColor  " Clicked="AddBorderColorButton_Clicked" VerticalOptions="Center" HorizontalOptions="Center" Grid.Row="0"/> 
+    <syncfusion:SfPdfViewer x:Name="PdfViewer" DocumentSource="{Binding PdfDocumentStream}" Grid.Row="1" /> 
+</Grid> 
+{% endhighlight %}
+
+{% highlight C# %}
+private void AddBorderColorButton_Clicked(object sender, EventArgs e) 
+{ 
+foreach (FormField formField in PdfViewer.FormFields)
+{
+    if (formField is TextFormField textBoxField)
+    {
+        foreach (var widget in textBoxField.Widgets)
+        {
+            widget. BorderColor = Colors.Blue;
+        }
+    }
+} 
+} 
+{% endhighlight %}
+{% endtabs %}
+
+# Customize BorderWidth for Form fields
+The BorderWidth sets the thickness of the border around a form field. This allows you to make the border thicker or thinner by modifying it in a PDF document programmatically.
+###  Customize BorderWidth for Form fields  programmatically 
+You can add a new form fields with Customize the  BorderWidth  the PDF document programmatically by creating an form fields 
+
+{% tabs %}
+{% highlight xaml %}
+<Grid> 
+    <Grid.RowDefinitions> 
+        <RowDefinition Height="Auto" /> 
+        <RowDefinition Height="*" /> 
+    </Grid.RowDefinitions> 
+    <Button Text="AddBorderWidth" Clicked="AddBorderWidthButton_Clicked" VerticalOptions="Center" HorizontalOptions="Center" Grid.Row="0"/> 
+    <syncfusion:SfPdfViewer x:Name="PdfViewer" DocumentSource="{Binding PdfDocumentStream}" Grid.Row="1" /> 
+</Grid> 
+{% endhighlight %}
+
+{% highlight C# %}
+private void AddBorderWidthButton_Clicked(object sender, EventArgs e) 
+{ 
+foreach (FormField formField in PdfViewer.FormFields)
+{
+    if (formField is TextFormField textBoxField)
+    {
+        foreach (var widget in textBoxField.Widgets)
+        {
+            widget. BorderWidth = 2.0f;
+        }
+    }
+} 
+} 
+{% endhighlight %}
+{% endtabs %}
+
+# PropertyChanged Event for Form fields
+The PropertyChanged event occurs when an property is changed in the PDF document. The following example explains how to wire and handle the event.
```

Comment:
Explain what is the specific use case you have handled in the code snippet as well